### ISSUE TITLE
Illuminations Mod and Iris Shader not working together

### ISSUE THREAD

**Issue by SugarDaddii** [IrisShaders/Iris#520]
### Iris and Minecraft Version

Iris v1.1.0 + Sodium for MC 1.17

### Are you running Sodium along with Iris?

Yes

### Operating System

Win 10

### What is your GPU?

GTX 1070

### Java Version

Java 16

### Reproduction Steps

1. Start Medival Minecraft Fabric Modpack with Iris Shader --> CRASHES
2. Deactivate Illuminations Mod --> Game starts normally

### Crash Report file and latest.log

```shell
Can give you Crash report because no log will be saved after GD Launcher say it was crashing
```
EDIT.: 
https://pastebin.com/Z9zmpVWw

**Comment by yitzy299** [comment]
So the illuminations mod seems to mixin to sodium, but due to modifications either in sodium 1.17 or iris sodium it cannot apply them

**Comment by IMS212** [comment]
This should be fixed by illuminations 1.6.
